tests/neg/given-pattern.scala

```diff
@@ -4,11 +4,11 @@ class Test {
   import scala.collection.immutable.{TreeSet, HashSet}
 
   def f2[T](x: Ordering[T]) = {
-    val (given y: Ordering[T]) = x
+    val (given Ordering[T]) = x
     new TreeSet[T]    // error: no implicit ordering defined for T
   }
   def f3[T](x: Ordering[T]) = {
-    val given y: Ordering[T] = x  // error: pattern expected
+    val given Ordering[T] = x
```

Comment:
Does this one work as a `val` definition in class?

Reply:
yes, it should work

Reply:
Then we should have some tests